.travis.yml

```diff
@@ -16,10 +19,34 @@ matrix:
   include:
     - php: '7.2'
       env: TARGET=behat
+    - php: '7.2'
+      env: TARGET=e2e
 
   allow_failures:
     - php: nightly
 
+before_install:
+  - |
+     if [[ ${TARGET} == "e2e" ]]; then
+       wget http://ftp.debian.org/debian/pool/main/libr/librabbitmq/librabbitmq-dev_0.5.2-2_amd64.deb
+       wget http://ftp.debian.org/debian/pool/main/libr/librabbitmq/librabbitmq1_0.5.2-2_amd64.deb
+       sudo dpkg -i librabbitmq1_0.5.2-2_amd64.deb librabbitmq-dev_0.5.2-2_amd64.deb
+
+       local ext_name=amqp-1.9.3
+       local ext_so=amqp.so
+       local ext_dir=$(php -r "echo ini_get('extension_dir');")
+       local ext_cache=~/php-ext/$(basename $ext_dir)/$ext_name
+
+       if [[ -e $ext_cache/$ext_so ]]; then
+           echo extension = $ext_cache/$ext_so >> ~/.phpenv/versions/7.2/etc/conf.d/travis.ini
```

Comment:
Can `TRAVIS_PHP_VERSION` be used here so this path becomes dynamic and doesn't couple to the 7.2 version?